src/components/reviews/Reviews.jsx

```diff
@@ -110,37 +115,74 @@ class Reviews extends React.Component {
         reviewsList.push(review);
       }
     }
-    const reviewCount = reviewsList.length;
-    const moreToLoad = newReviews.length === this.state.reviewsPerLoad;
+    const reviewsLoaded = reviewsList.length;
+    const moreToLoad = !(newReviews.length < this.state.reviewsPerLoad);
     const filter = this.state.filter;
     const filteredReviewsList = [];
     for (let review of reviewsList) {
       if (filter(review)) {
         filteredReviewsList.push(review);
       }
     }
+    filteredReviewsList.sort(this.getSortFunction());
     this.setState(
-      { reviewsList, filteredReviewsList, reviewCount, moreToLoad, isLoading: false },
+      { reviewsList, filteredReviewsList, reviewsLoaded, moreToLoad, isLoading: false },
       callback
     );
   }
 
+  getSortFunction() {
+    const { sortOrder } = this.state;
+    if (sortOrder === 'newest') {
+      return (a, b) => {
+        if (new Date(a.date) < new Date(b.date)) {
+          return 1;
+        } else {
+          return -1;
+        }
+      };
+    } else if (sortOrder === 'relevant') { // not sure how to sort for relevancy?
```

Comment:
According to the business doc, I think the relevant option can be sorted by a combination of both the date that review was submitted and the number of helpful points.